static/docs/commands-reference/push.md

```diff
@@ -24,7 +24,9 @@ environments and preserving data versions (input datasets, intermediate results,
 models, metrics, etc) remotely (S3, SSH, GCS, etc) are the most common use cases
 for these commands.
 
-The `dvc push` command allows one to upload data to remote storage.
+The `dvc push` command allows one to upload data to remote storage. It is run
+along with 'git commit' and
+[git push](https://git-scm.com/docs/git-push/1.5.5.2).
```

Comment:
let's remove the link, it looks weird that we are linking only git push. And it should be clear to ppl what does git push mean

Reply:
okay! Shall i explain ``git push`` instead?